src/library-configuration/ethereum-providers.md

```diff
@@ -0,0 +1,58 @@
+# Ethereum Providers
+
+Resolution libraries require a connection to the Ethereum network to resolve .crypto and .eth domains. To initialize the library, you need to specify an Ethereum node service provider. Once the instance is created you can begin resolving domains. Below are examples of how to initialize the library with different providers.
+
+### Import library
+
+Javascript:
+
+```javascript
+import Resolution from "@unstoppabledomains/resolution";
+```
+
+### Infura
+
+[Infura]("https://infura.io") is a popular Ethereum node service. Obtaining an API key is free and only requires creating an account.
+
+Javascript:
+
+```javascript
+const infuraApiKey = INFURA_PROJECT_ID;
+
+const resolution = new Resolution({
+  cns: {
+    url: `https://mainnet.infura.io/v3/${infuraApiKey}`,
+    network: "mainnet",
+  },
+});
+
+// or
+
+const resolution = Resolution.infura(infuraApiKey, "mainnet");
+```
+
+### Web3Provider
```

Comment:
@sudoryan Let's go with one config per library. The intent of the article to show how to connect with external Ethereum providers like Infura. 

Other methods of configuration below can be moved to the library README.